tensorflow/python/ops/math_ops.py

```diff
@@ -334,7 +334,32 @@ def divide(x, y, name=None):
 
 @tf_export("math.multiply", "multiply")
 @dispatch.add_dispatch_support
+
+def multiply(x, y, name=None):  # pylint: disable=missing-docstring
```

Comment:
I don't understand why you need lines 338-342 here (that is, why are there two "def multiply")

Reply:
Sorry about that! I will make changes